src/XIVLauncher.Common.Unix/UnixDalamudRunner.cs

```diff
@@ -74,43 +74,57 @@ public UnixDalamudRunner(CompatibilityTools compatibility, DirectoryInfo dotnetR
         launchArguments.Add(gameArgs);
 
         var dalamudProcess = compatibility.RunInPrefix(string.Join(" ", launchArguments), environment: environment, redirectOutput: true, writeLog: true);
-        var output = dalamudProcess.StandardOutput.ReadLine();
+        var output = dalamudProcess.StandardOutput.ReadLine() ?? "";
 
-        if (output == null)
-            throw new DalamudRunnerException("An internal Dalamud error has occured");
+        Console.WriteLine("[DALAMUD] " + output);
+
+        // Skip "ERROR: Could Not Get Primary Adapter Handle" and proceed to next line
+        if (output.Equals("ERROR: Could Not Get Primary Adapter Handle"))
```

Comment:
Why is this necessary?
Wine should always output its internal logging to stderr, but this doesn't even look like wine.